Lib/python/builtin.swg

```diff
@@ -211,7 +211,11 @@ SwigPyStaticVar_Type(void) {
       sizeof(PyGetSetDescrObject),              /* tp_basicsize */
       0,                                        /* tp_itemsize */
       (destructor)SwigPyStaticVar_dealloc,      /* tp_dealloc */
-      0,                                        /* tp_print */
+#if PY_VERSION_HEX < 0x030800b4
+      (printfunc)0,                             /*tp_print*/
+#else
+      (Py_ssize_t)0,                            /*tp_vectorcall_offset*/
+#endif
```

Comment:
These comments (and those below) are inconsistently formatted compared to the existing comments above and below them which all have spaces around the content (which also seems better as it's easier to read).

Reply:
should be fixed now